### PR TITLE
Use hexadecimal sizes in table visitor

### PR DESCRIPTION
**Pull request by rjoleary**
Additionally, only print the FV free space when the free space is not
empty.

Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

**Comment by rjoleary** [comment]
Fixes #168

File: pkg/visitors/table.go [hunk 1]
```diff
@@ -28,50 +28,50 @@ func (v *Table) Run(f uefi.Firmware) error {
...
 
-func (v *Table) printRow(f uefi.Firmware, node, name, typez, size interface{}) error {
+func (v *Table) printRow(f uefi.Firmware, node, name, typez interface{}) error {
 	if v.W == nil {
 		v.W = tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
 		defer func() { v.W.Flush() }()
 		fmt.Fprintf(v.W, "%sNode\tGUID/Name\tType\tSize\n", indent(v.indent))
 	}
-	fmt.Fprintf(v.W, "%s%v\t%v\t%v\t%v\n", indent(v.indent), node, name, typez, size)
+	fmt.Fprintf(v.W, "%s%v\t%v\t%v\t%#8x\n", indent(v.indent), node, name, typez, len(f.Buf()))
```
Comment (GanShun) [inline pkg/visitors/table.go]: At some point we should make Size() be a function that checks the length of the buffer, and SetBuf reset the size.

File: pkg/visitors/table.go [hunk 2]
```diff
@@ -28,50 +28,50 @@ func (v *Table) Run(f uefi.Firmware) error {
...
 	v2 := *v
 	v2.indent++
 	if err := f.ApplyChildren(&v2); err != nil {
 		return err
 	}
-	if fv, ok := f.(*uefi.FirmwareVolume); ok {
+	if fv, ok := f.(*uefi.FirmwareVolume); ok && fv.FreeSpace != 0 {
```
[the remark this answers is not in the document]
Comment (rjoleary) [reply pkg/visitors/table.go]: Done, I reverted it

**Comment by rjoleary** [comment]
PTAL

File: pkg/visitors/table.go (hunk 1 above)
Comment (GanShun) [reply pkg/visitors/table.go]: sort of. We need to also figure out how to always get the correct size from Length(), as what's underlying may have changed.

**Pull request merged**